### ISSUE TITLE
The old extension setting for enabling Sequence Diagram support is removed

### ISSUE THREAD

**Issue by brikelly** [getappmap/vscode-appland#618]

**Comment by ahtrotta** [comment]
@brikelly , do you mean this setting?



![Image](https://user-images.githubusercontent.com/45714532/223209041-c641e415-d71c-4a43-a445-83a8e2fd2bb6.png)

**Comment by brikelly** [comment]
Yes, that's the one - but we can't remove it until we have feature equivalence with the PlantUML stuff. So we still need sequence diffs before we can remove this, basically.
